BlueprintUI/Sources/Layout/Builder.swift

```diff
@@ -42,6 +42,18 @@ public struct Builder<Child> {
         fatalError()
     }
 
+    public static func buildArray(_ components: [Children]) -> Children {
+        components.reduce(into: []) { $0 += $1 }
```

Comment:
I think this might be a little more efficient:
```suggestion
        components.flatMap { $0 }
```

Reply:
I tested this and didn't find any particularly large difference

Reply:
Yeah I'm sure its not too much different, especially on arrays with such few elements. I do think `flatMap` is more semantically appropriate in this scenario since we are flattening a two dimensional array into a single level collection which jives with Apple's [documentation](https://developer.apple.com/documentation/swift/sequence/2905332-flatmap). But its not so important to me that I'd debate it any more than writing this post 😆

Reply:
True, I can switch it up